lib/Ed25519VerificationKey2020.js

```diff
@@ -234,6 +284,58 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
     return exportedKey;
   }
 
+  /**
+   * Returns the JWK representation of this key pair.
+   *
+   * @see https://datatracker.ietf.org/doc/html/rfc8037
+   *
+   * @param {object} [options={}] - Options hashmap.
+   * @param {boolean} [options.publicKey] - Include public key?
+   * @param {boolean} [options.privateKey] - Include private key?
+   *
+   * @returns {{kty: string, crv: string, x: string, d: string}} JWK
+   *   representation.
+   */
+  toJwk({publicKey = false, privateKey = false} = {}) {
+    const jwk = {crv: 'Ed25519', kty: 'OKP'};
+    if(publicKey) {
+      jwk.x = base64url.encode(this._publicKeyBuffer);
+    }
+    if(privateKey) {
+      jwk.d = base64url.encode(this._privateKeyBuffer);
+    }
+    return jwk;
+  }
+
+  /**
+   * @see https://datatracker.ietf.org/doc/html/rfc8037#appendix-A.3
+   *
+   * @returns {Promise<string>} JWK Thumbprint.
+   */
+  async jwkThumbprint() {
+    const serialized = JSON.stringify(this.toJwk({publicKey: true}));
```

Comment:
Does this need to use `canonicalize`?  The example in that RFC is sorting keys from previous examples.  I'm not sure what current guarantees are from `JSON.stringify`.

Reply:
@davidlehn - good call. I refactored it so that it doesn't use json.stringify.